src/js/control-bar/control-bar.js

```diff
@@ -56,7 +55,8 @@ class ControlBar extends Component {
 ControlBar.prototype.options_ = {
   children: [
     'playToggle',
-    'volumeMenuButton',
+    'muteToggle',
```

Comment:
what about instead of making them direct children of the control bar have a second component that has them as children? That way, we can style them better as part of the accessible control that works like the volume-menu-button but also separately, for those that want them separate.